framework/src/transfers/MultiAppGeneralFieldShapeEvaluationTransfer.C

```diff
@@ -136,6 +138,10 @@ MultiAppGeneralFieldShapeEvaluationTransfer::evaluateInterpValuesWithMeshFunctio
           const Point nearest_position = _nearest_positions_obj->getNearestPosition(pt, initial);
           nearest_position_source =
               _nearest_positions_obj->getNearestPosition(_from_positions[i_from], initial);
+          if (!_skip_coordinate_collapsing &&
+              _from_transforms[from_global_num]->hasNonTranslationTransformation())
+            mooseError(
+                "Rotation and scaling currently unsupported with nearest positions transfer.");
```

Comment:
This is a `ShapeEvaluationTransfer`

Reply:
but maybe this message is still correct

Reply:
yup. nearest-positions can still be done for shapeEvaluation + overlapping domains. 
Howver it s kind of a stretch